halo/Dockerfile

```diff
@@ -8,6 +8,8 @@ COPY --from=alpine:latest /tmp /tmp
 
 COPY halo /app
 
+HEALTHCHECK --interval=30s --timeout=5s --start-period=10s CMD ["/app", "ready"]
```

Comment:
I'd just stick to default values, why do we need to change timeout or start period?

Reply:
True, it helped me debugging it, but the default ones seem to be good enough for production, epscially since this is not really used anywhere yet.